.github/workflows/python-package.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest  # Alternative: windows-latest
     strategy:
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10']  # For multiple: [3.7, 3.8]
+        python-version: [3.7, 3.8, 3.9, '3.10', 3.11]
```

Comment:
Only the "package" should be run on them all - the rest we can just run on a single one - but maybe move it to Python 3.11 or similar.